common/persistence/pinot/pinotVisibilityStore.go

```diff
@@ -743,15 +741,6 @@ func (f *PinotQueryFilter) addTimeRange(obj string, earliest interface{}, latest
 	f.string += fmt.Sprintf("%s BETWEEN %v AND %v\n", obj, earliest, latest)
 }
 
-func (f *PinotQueryFilter) addPartialMatch(key string, val string) {
-	f.checkFirstFilter()
-	f.string += fmt.Sprintf("%s\n", getPartialFormatString(key, val))
-}
-
-func getPartialFormatString(key string, val string) string {
-	return fmt.Sprintf(LikeStatement, key, val)
-}
-
```

Comment:
to be consistent, let's rename the files in the PR to be camel_case. e.g. pinot_visibility_store_test.go